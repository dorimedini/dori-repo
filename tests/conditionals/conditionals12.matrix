{
if(false)
if(true)
print "Fail.";
else
print "Fail.";
else
print "Ok.";
print "Should not print fail...";
}
