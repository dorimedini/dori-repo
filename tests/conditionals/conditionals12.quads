goto 6
goto 2
printLn "Fail."
goto 7
printLn "Fail."
goto 7
printLn "Ok."
printLn "Should not print fail..."
