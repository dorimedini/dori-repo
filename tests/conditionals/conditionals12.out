Ok.
Should not print fail...
